railties/lib/rails/generators/rails/resource/resource_generator.rb

```diff
@@ -13,7 +13,7 @@ class ResourceGenerator < ModelGenerator # :nodoc:
       end
 
       class_option :actions, type: :array, banner: "ACTION ACTION", default: [],
-                             desc: "Actions for the resource controller"
+                             desc: "Action for the resource controller"
```

Comment:
This is currently correct as Actions is a noun here.
```suggestion
                             desc: "Actions for the resource controller"
```